src/components/aboutPage/OurStory.jsx

```diff
@@ -7,8 +7,11 @@ export default function OurStory() {
         <h1 className='mt-[37px] md:mt-[126px] mb-[14px] md:mb-[43px] text-[32px] md:text-[64px] text-[#025594] font-bold leading-[38px] md:leading-[75px]'>Our Story</h1>
         </div>
         <div className='mb-[50px] md:mb-[129px]'>
-         <p className='max-w-[312px] md:max-w-[817px] text-center text-[16px] md:text-[24px] font-normal leading-[18.75px] md:leading-[28px]'>
-         At RefuBook, we understand the challenges that come with being a refugee and that is why we are dedicated to providing comprehensive support and resources to individuals who have been forced to flee their homes due to conflict, persecution, or other life-threatening circumstances. We believe that every refugee deserves the opportunity to thrive in their new surroundings. Through our platform, we connect refugees with resources and support, such as housing, medical care, education, and employment. We also work to advocate for the rights and needs of refugees on all levels, locally, nationally and internationally. But we can not do it alone, with your help, by volunteering, donating, spreading awareness, or simply welcoming new refugees in your community, you can make a meaningful difference in the lives of refugees. Join us in creating a more inclusive and welcoming world for all.
+         <p className='max-w-[312px] md:max-w-[817px] text-center text-[16px] md:text-[24px] font-normal leading-4 md:leading-[36px]'>
+         At RefuBook, we understand the challenges that come with being a refugee and that is why we are dedicated to providing comprehensive support and resources to individuals who have been forced to flee their homes due to conflict, persecution, or other life-threatening circumstances. We believe that every refugee deserves the opportunity to thrive in their new surroundings. 
+         <br /> 
+         <br />
+         Through our platform, we connect refugees with resources and support, such as housing, medical care, education, and employment. We also work to advocate for the rights and needs of refugees on all levels, locally, nationally and internationally. But we can not do it alone, with your help, by volunteering, donating, spreading awareness, or simply welcoming new refugees in your community, you can make a meaningful difference in the lives of refugees. Join us in creating a more inclusive and welcoming world for all.
```

Comment:
in mobile view, text shouldn't be look like this,  give w-full  e.i
![chrome_6NHt0TfSz4](https://user-images.githubusercontent.com/92088301/211835150-4cef50ff-2d12-4079-afbf-61861868b64d.png)

![chrome_jiV6jHZinj](https://user-images.githubusercontent.com/92088301/211835141-617765c6-2b48-4b56-9df9-1e5ae9eb0732.png)